### PR TITLE
Updating Querqy documentation for OS 2.3

### PR DESCRIPTION
**Pull request by ariamarble**
Signed-off-by: ariamarble <armarble@amazon.com>

### Description
Updates Querqy documentation for OS 2.3

### Issues Resolved
[#1054](https://github.com/opensearch-project/project-website/issues/1054) 

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by ariamarble** [comment]
tech review complete

**Review by JeffH-AWS — APPROVED** [review]
Looks good!

File: _search-plugins/querqy/index.md [hunk 1]
```diff
@@ -13,51 +13,34 @@ Querqy is a community plugin for query rewriting that helps to solve relevance i
 
 ## Querqy plugin installation
 
-Querqy is currently only compatible with OpenSearch 1.3.1
-{: .note }
+The Querqy plugin is now available for OpenSearch 2.3.0. Run the following command to install the Querqy plugin.
```
[the remark this answers is not in the document]
Comment (ariamarble) [reply _search-plugins/querqy/index.md:16]: 2.3 and higher

[the remark this answers is not in the document]
Comment (ariamarble) [reply _search-plugins/querqy/index.md:16]: 1.3 is compatible with the old install which is why I'm only back-porting to 2.3

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**